### PR TITLE
feat: Image Widget Reskinning

### LINKED ISSUE #10979: [Enhancement]: Image widget reskinning
https://www.notion.so/appsmith/Image-ba96edc31359457789588d7ada8b0556

### PR DESCRIPTION
**Pull request by dhruvikn**
## Description

This PR fixes UI inconsistencies in the Image widget.

- Update rotate and download control icons
- Add border radius and box shadow to control wrapper
- Multiple CSS fixes to follow the designs

Fixes #10979

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
- Manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/widgets/ImageWidget/component/index.tsx [hunk 1]
```diff
@@ -265,67 +290,69 @@ class ImageComponent extends React.Component<
...
+              >
+                <svg
+                  fill="none"
+                  height="12"
+                  viewBox="0 0 12 12"
+                  width="12"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M2.23333 2.17702C3.19999 1.21035 4.52666 0.610352 5.99999 0.610352C8.94666 0.610352 11.3267 2.99702 11.3267 5.94368C11.3267 8.89035 8.94666 11.277 5.99999 11.277C3.51333 11.277 1.43999 9.57702 0.846661 7.27702H2.23333C2.77999 8.83035 4.25999 9.94368 5.99999 9.94368C8.20666 9.94368 9.99999 8.15035 9.99999 5.94368C9.99999 3.73702 8.20666 1.94368 5.99999 1.94368C4.89333 1.94368 3.90666 2.40368 3.18666 3.13035L5.33333 5.27702H0.66666V0.610352L2.23333 2.17702Z"
```
Comment (keyurparalkar) [inline app/client/src/widgets/ImageWidget/component/index.tsx:308]: @dhruvikn can you please move this SVG to a folder?

File: app/client/src/widgets/ImageWidget/component/index.tsx [hunk 2]
```diff
@@ -265,67 +290,69 @@ class ImageComponent extends React.Component<
...
-                    <path
-                      d="M0.400024 6.00039C0.400024 2.91317 2.91281 0.400391 6.00002 0.400391C7.42515 0.400391 8.72559 0.939565 9.71513 1.81862L10.6667 0.867057V3.66706H7.86669L9.05432 2.47943C8.23558 1.7682 7.17228 1.33372 6.00002 1.33372C3.41724 1.33372 1.33336 3.41761 1.33336 6.00039C1.33336 8.58317 3.41724 10.6671 6.00002 10.6671C8.58281 10.6671 10.6667 8.58317 10.6667 6.00039H11.6C11.6 9.08761 9.08724 11.6004 6.00002 11.6004C2.91281 11.6004 0.400024 9.08761 0.400024 6.00039Z"
-                      fill="#858282"
-                      stroke="#858282"
-                      strokeWidth="0.5"
-                    />
-                  </svg>
-                </div>
+                <svg
+                  fill="none"
```
Comment (keyurparalkar) [inline app/client/src/widgets/ImageWidget/component/index.tsx:318]: Move svg to different folder

File: app/client/src/widgets/ImageWidget/component/index.tsx [hunk 3]
```diff
@@ -265,67 +290,69 @@ class ImageComponent extends React.Component<
...
-                  <path
-                    clipRule="evenodd"
-                    d="M15.4547 16.4284H13.117H6.88326H4.54559C2.8243 16.4284 1.42871 14.8933 1.42871 12.9999C1.42871 11.3987 2.43157 10.0641 3.7804 9.68786C3.93001 6.28329 6.47884 3.57129 9.61053 3.57129C12.7072 3.57129 15.2349 6.22243 15.4352 9.57386C17.183 9.56015 18.5716 11.1167 18.5716 12.9999C18.5716 14.8933 17.176 16.4284 15.4547 16.4284ZM12.7266 11.4286L9.99929 14.8572L7.27202 11.4286L8.83045 11.4286L8.83045 8.00004L11.1681 8.00003V11.4286L12.7266 11.4286Z"
-                    fill="#939090"
-                    fillRule="evenodd"
-                  />
-                </svg>
-              </div>
+              <svg
+                className="is-download-icon"
```
Comment (keyurparalkar) [inline app/client/src/widgets/ImageWidget/component/index.tsx:344]: Move svg to different folder

**Review by keyurparalkar — COMMENTED** [review]
Whenever the object is fit to cover and then rotated then should the border-radius be applied?
Screenshot 2022-09-15 at 4 42 07 PM

Also, seems like the rotate icon is bit off from center?
Screenshot 2022-09-15 at 4 41 27 PM

**Comment by dhruvikn** [comment]
Hey @keyurparalkar, the rotate icon is centered. 😄 
I took some screenshots, and applied constraints of the same dimensions around the icon. Please check this [Figma](https://www.figma.com/file/DvUd9OJqhMfNgXXLE21PN7/Rotate-Icon-Center-Checks?node-id=0%3A1) or [PDF](https://github.com/appsmithorg/appsmith/files/9581534/Rotate.Icon.Center.Checks.pdf) out with maximum zoom in level.

image

Moreover, for this -

> Whenever the object is fit to cover and then rotated then should the border-radius be applied?

Had a discussion with @dilippitchika, there are a few thoughts - 
1. We have changed the deafult object fit to cover now.
2. The border radius issue when the image is rotated still exists. Border radius also fails when we change the dimensions of the image. This happens because we are rendering images as a `div` background and we are applying border radius to the container. This can be solved if we render images inside an `` tag. This is a larger problem and we will be creating a new issue for the same.

**Review by keyurparalkar — APPROVED** [review]
LGTM!

File: app/client/src/widgets/ImageWidget/component/index.tsx [hunk 4]
```diff
@@ -51,46 +54,69 @@ const Wrapper = styled.div<{
...
+    svg path {
+      fill: ${Colors.GREY_9};
     }
   }
 `;
 
+const Separator = styled.div`
+  height: 18px;
+  width: 1px;
+  background-color: ${Colors.GREY_5};
```
Comment (jsartisan) [inline app/client/src/widgets/ImageWidget/component/index.tsx:116]: Hey can we use the css variable here?

**Review by jsartisan — CHANGES_REQUESTED** [review]
LGTM. Just one comment.

File: app/client/src/widgets/ImageWidget/component/index.tsx (hunk 4 above)
Comment (dhruvikn) [reply app/client/src/widgets/ImageWidget/component/index.tsx:116]: Do you mean variables for colors from here? https://github.com/appsmithorg/appsmith/blob/769719ccfe667f059fe0b107a19ec9feb90f2e40/app/client/src/theme/wds.css

**Comment by laveena-en** [comment]
Tested and verified.

**Review by jsartisan — APPROVED** [review]

**Review by keyurparalkar — APPROVED** [review]

**Pull request merged**